terraform_backend/backend.tf

```diff
@@ -6,11 +6,10 @@ variable "aws_region" {
   default = "eu-west-2" # london
 }
 
-variable "s3_app_name" {}
+variable "backend_bucket" {}
```

Comment:
Is this used?

Reply:
not any more! forgot to delete it after I baked app_name into the shared bucket